.cbmc-batch/source/make_common_data_structures.c

```diff
@@ -114,6 +114,21 @@ bool aws_cryptosdk_edk_list_elements_are_bounded(const struct aws_array_list *co
     return true;
 }
 
+bool ensure_cryptosdk_edk_list_elements_are_readable(struct aws_array_list *list) {
```

Comment:
normally `ensure` functions allocate memory. This seems like an `is_valid` type function?

Reply:
You're right! I've actually discovered a aws_cryptosdk_edk_list_elements_are_valid function which I am now using instead.